### DISCUSSION TITLE
How to upload Android app source code to Github account via github.com

### DISCUSSION THREAD

**Discussion by pinknblue22** [community/community#39862] [New to GitHub]
Hello,

I am new to github and coding.

how do I upload my android app source code folder to my github account so we may keep track of versions and who has access?

Thanks

**Comment by ossanna16** [comment]
Hi there @pinknblue22 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
